docs/.vuepress/theme/components/events/Events.vue

```diff
@@ -25,20 +25,36 @@
         <div class="upcoming-event-list">
           
           <div class="event-card">
-            <img src="./workshop.svg" alt="Landing Zone Workshop" />
+            <img src="./LiveDemo_Steph.webp" alt="Live Demo" />
+            <div class="event-card__content">
+              <h4>meshStack Live Demo (livestream) - March 13th, 14:00 - 15:00 (CET) </h4>
+              <p>
+                Tune in to our meshStack live demo on YouTube to learn more about meshStack's capabilities and see it in action.
+                Subscribe to our channel to stay updated.
+              </p>
+              <a
+                href="https://www.meshcloud.io/en/company/events/live-demo/"
+                class="btn btn-outline-primary btn-lg"
+              >
+                Subscribe!
+              </a>
+            </div>
+          </div>
+
+          <div class="event-card">
+            <img src="./EA_Workshop.webp" alt="Clean up Your Cloud Workshop" />
```

Comment:
```suggestion
            <img src="./EA_Workshop.webp" alt="Clean up Your Cloud Workshop" style="border-radius: 45px;"/>
```

If you add this, we get nice rounded corners for this picture too :-)

<img width="482" alt="image" src="https://github.com/cloudfoundation-community/cloudfoundation.org/assets/14248574/9fed1ad4-352e-4fe3-8de4-69991ceaef60">